setup.rst

```diff
@@ -64,12 +64,12 @@ Symfony application using Composer:
 .. code-block:: terminal
 
     # run this if you are building a traditional web application
-    $ composer create-project symfony/skeleton:"^6.1" my_project_directory
+    $ composer create-project symfony/skeleton:"^6.2" my_project_directory
```

Comment:
Same question  as on your other PR: Should we use `6.2*` here?

Reply:
Yes, I re-edited this